### PR TITLE
SL-20031: Stop running Windows wmic; use PyPI wmi package instead.

### PR DESCRIPTION
**Pull request by nat-goodspeed**
Remove wmic() and _wmic() functions, also WmicError.

Adjust tests accordingly. Delete test intentionally raising WmicError.

Also remove cruft concerning 32-bit Python.

### CODE REVIEW

File: build-cmd.py [hunk 1]
```diff
@@ -120,40 +120,15 @@ def main():
     with suppress(FileExistsError):
         os.makedirs(stage)
 
-    #We ship a 32 bit VMP with 64 bit viewers
-    if system() == 'Windows' and getAddressSize() == 64:
```
Comment (akleshchev) [inline build-cmd.py:124]: So it's going to be 64 only? What branch do you plan to put this into? If it's going to be 64 only, we will have to adjust 32 bit viewer to try to call updater, but not complain about missing updater.

**Review by akleshchev — APPROVED** [review]

File: build-cmd.py (hunk 1 above)
Comment (nat-goodspeed) [reply build-cmd.py:124]: We've officially dropped support for 32-bit viewers. I'm hoping to merge into Maint V, but I'm wrestling with TC Windows builds of viewer-manager, so not yet.

Comment (akleshchev) [reply build-cmd.py:124]: Is there a point to keep 'hasOnlyUnsupported' part of gpu detection? Is 32 bit driver problem relevant to Win 10? Afected gpus Intel HD 2000/3000 are not supported on Win 10. Win7 and Win 8.1 no longer receive support from MS.

Comment (nat-goodspeed) [reply build-cmd.py:124]: But as long as the 32-bit Maint S viewer will still run on Windows 10+, I believe there's still a point to downgrading a user with Intel HD 2000/3000 to that viewer.

**Comment by nat-goodspeed** [comment]
This branch is a strict subset of DRTVWR-587-actions.

**Pull request closed without merging**